fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/LocalResourceReferenceRec.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.dao.impl;
+
+
+/**
+ * A DTO representing an external resource reference. This object is initialized
+ * with names which are resolved to ids when passed through the cache. The idea
+ * is to reduce the number of times we have to iterate over the list of records
```

Comment:
Local not External?

Reply:
This probably is no longer required, but too risky to remove at this point so creating an issue to cover clean-up.

Reply:
My comment is to the DTO comment is on external, the class cites Local 
Is this comment accurate?

Reply:
Doesn't matter. The design changed so this class is no longer required. Issue #1639 for cleanup.